### PR TITLE
kgo: use helm as installation method

### PR DESCRIPTION
**Pull request by pmalek**
### Description

Rewrites KGO installation guides to all use helm and KGO's helm chart.

Removes the old instructions which used the manifests.

Related: https://github.com/Kong/gateway-operator-archive/issues/1594

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

File: app/_src/gateway-operator/get-started/konnect/install.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ book: kgo-konnect-get-started
...
 
-{{ site.kgo_product_name }} can deploy and manage data planes attached to a {{ site.konnect_short_name }} control plane. All the services, routes, and plugins are configured in {{ site.konnect_short_name }} and sent to the data planes automatically.
-
-{% include snippets/gateway-operator/install_with_kubectl.md %}
+{{ site.kgo_product_name }} can deploy and manage data planes attached to a {{ site.konnect_short_name }} control plane.
+All the services, routes, and plugins are configured in {{ site.konnect_short_name }} and sent to the data planes automatically.
 
 Once the `gateway-operator-controller-manager` deployment is ready, you can deploy a `DataPlane` resource that is attached to a {{ site.konnect_short_name }} control plane.
+
+{% include md/kgo/prerequisites-kic.md disable_accordian=true version=page.version release=page.release %}
```
Comment (mheap) [inline app/_src/gateway-operator/get-started/konnect/install.md:13]: Should this just be `prerequisites.md` if there's nothing specific to KIC?

File: app/_data/docs_nav_kgo_1.2.x.yml [hunk 2]
```diff
@@ -54,8 +54,6 @@ items:
     items:
       - text: Overview
         url: /production/
-      - text: Install
```
Comment (mheap) [inline app/_data/docs_nav_kgo_1.2.x.yml:57]: We should keep the standalone install page. It's one that many people look for when visiting for the first time

**Review by mheap — CHANGES_REQUESTED** [review]

File: app/_src/gateway-operator/get-started/konnect/install.md (hunk 1 above)
Comment (pmalek) [reply app/_src/gateway-operator/get-started/konnect/install.md:13]: Makes sense, let me change that.

File: app/_data/docs_nav_kgo_1.2.x.yml (hunk 2 above)
Comment (pmalek) [reply app/_data/docs_nav_kgo_1.2.x.yml]: Got it, reverted that change.

**Review by mheap — APPROVED** [review]

**Pull request merged**